app/src/main/java/com/vero/tasky/core/data/local/PreferencesImpl.kt

```diff
@@ -0,0 +1,28 @@
+package com.vero.tasky.core.data.local
+
+import android.app.Application
+import android.content.Context
+import androidx.core.content.edit
+import com.vero.tasky.core.domain.local.Preferences
+import javax.inject.Inject
+
+class PreferencesImpl @Inject constructor(
+    application: Application
+) : Preferences {
+
+    private val pref = application.getSharedPreferences(
+        "com.vero.tasky", Context.MODE_PRIVATE)
+
+    override fun isLoggedIn() = getJwtToken() != null
+
+    override fun getJwtToken(): String? {
```

Comment:
When a user logs in, you get user token, email and full name

You should store all of that (will be used in the future)